Loop/View Controllers/StatusTableViewController.swift

```diff
@@ -205,16 +205,14 @@ class StatusTableViewController: UITableViewController, UIGestureRecognizerDeleg
                 }
             }
 
-            reservoirVolume = dataManager.latestReservoirValue?.unitVolume
```

Comment:
This (class property) needs to be set here before its referenced below on line 209